docs/source/community/issue_tracking.rst

```diff
@@ -35,14 +35,16 @@ for initial feedback before submitting the issue in JIRA.
 JIRA
 ====
 
-We use JIRA as our issue tracking system.  The JIRA instance is hosted by
-the Hyperledger Project, and Hyperledger Sawtooth uses the key *STL*:
+Hyperledger Sawtooth uses JIRA as our issue tracking system, found using this
+`Link <https://jira.hyperledger.org/secure/RapidBoard.
+jspa?rapidView=3&projectKey=STL&view=planning.nodetail>`_
 
-  https://jira.hyperledger.org/projects/STL/issues/
+If you want to to contribute to Hyperledger Sawtooth quickly, we have a list of
+issues which will help you get involved right away. This can be found
+`here <https://jira.hyperledger.org/issues/
+?filter=10612>`_
 
-To report issues, you will need to login to JIRA, which requires a
-Linux Foundation account.  To setup a Linux Foundation account, go to:
-
-  https://identity.linuxfoundation.org/
+To report issues, you will need to login to Hyperledger Sawtooth JIRA, which requires a
```

Comment:
"login" is a noun. Use "log in" for the verb form you mean, e.g.:
"To report issues, you will need to log in to Hyperledger Sawtooth JIRA."
You could also use "sign in".